### PR TITLE
Report the inference benchmark of models with different size

### PR DESCRIPTION
**Pull request by lvhan028**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: benchmark/benchmark-7b.sh [hunk 1]
```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+if [ - z "$1" ]
+then
+    echo
+fi
+
+tp=1
+model_name=llama2
+model_path=/workspace/models-140/llama2/huggingface/llama-2-7b-chat/
```
Comment (zhulinJulia24) [inline benchmark/benchmark-7b.sh:9]: 能否将model_path和turbomind_model_path作为shell的入参

File: benchmark/benchmark-7b.sh
Comment (zhulinJulia24) [inline benchmark/benchmark-7b.sh:1]: filename benchmark_7b.sh 和别的文件可以对齐

File: benchmark/benchmark-7b.sh [hunk 2]
```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+if [ - z "$1" ]
```
Comment (zhulinJulia24) [inline benchmark/benchmark-7b.sh:2]: - z 需要改成 -z

File: benchmark/benchmark-7b.sh [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    output_path=$1
+    mkdir -p "${output_path}"
+
+    batches=(64 128)
+    for batch in "${batches[@]}"
+    do
+        for i in {1..3}
+        do
+        python3 benchmark/profile_throughput.py \
+            benchmark/ShareGPT_V3_unfiltered_cleaned_split.json \
```
Comment (zhulinJulia24) [inline benchmark/benchmark-7b.sh:37]: 这个文件要不要在我们的仓库也上传下，否则会找不到

File: benchmark/benchmark_13b.sh [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+apt-get install crudini
+
+crudini --set ${config_path} llama max_context_token_num 4
+crudini --set ${config_path} llama cache_chunk_size -1
+crudini --set ${config_path} llama cache_max_entry_count 500
+crudini --set ${config_path} llama max_batch_size 128
+# end of update config
+
+benchmark_rpm () {
+    output_path=$1
```
Comment (zhulinJulia24) [inline benchmark/benchmark_13b.sh:23]: 没有判断$1的入参

File: docs/en/benchmark/a100_fp16.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
+# Benchmark on A100 (FP16)
+
+All the following results are tested on (x8) A100-80G CUDA 11.8.
+
+The tested lmdeploy version is `v0.1.0a1`.
+
+The commands provided below facilitate benchmarking both [static inference performance](#static-inference-benchmark) and [request throughput](#request-throughput-benchmark) on an A100-80G(x8) for models of various sizes.
+
+```shell
```
Comment (zhulinJulia24) [inline docs/en/benchmark/a100_fp16.md:9]: 使用时需要传入output path

File: benchmark/benchmark-7b.sh [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+turbomind_model_path=workspace/llama2-7b-chat
+foldername=$(basename "$turbomind_model_path")
+
+# convert
+lmdeploy convert ${model_name} ${model_path} --dst-path ${turbomind_model_path} --tp ${tp}
+
+# update recommended config to config.ini
+config_path=${turbomind_model_path}/triton_models/weights/config.ini
+
+apt-get install crudini
```
Comment (zhulinJulia24) [inline benchmark/benchmark-7b.sh:19]: 是否使用requirement更好，可能存在系统没有apt-get

File: benchmark/benchmark_13b.sh (hunk 4 above)
Comment (lvhan028) [reply benchmark/benchmark_13b.sh:23]: 这个脚本内部的函数，不是对外接口。调用者自己传入就好

File: benchmark/benchmark-7b.sh (hunk 3 above)
Comment (lvhan028) [reply benchmark/benchmark-7b.sh:37]: 不放在仓库里。我在脚本中提供 wget 下载吧

File: benchmark/benchmark_13b.sh [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+python3 lmdeploy/turbomind/generate_gemm_config.py \
+    --head_num ${head_num} \
+    --size_per_head ${size_per_head} \
+    --vocab_size ${vocab_size} \
+    --inter_size ${inter_size} \
+    --tensor_para_size ${tensor_para_size} \
+    --max_batch_size ${max_batch_size}
+
+output_path=benchmark/output/"${foldername}"-tunned-gemm-tp"${tp}"
```
Comment (zhulinJulia24) [inline benchmark/benchmark_13b.sh:70]: output_path=$1 不生效

File: benchmark/benchmark-7b.sh (hunk 6 above)
Comment (lvhan028) [reply benchmark/benchmark-7b.sh]: requirement是通过 pip install 安装，不是 apt-get

File: benchmark/benchmark_13b.sh (hunk 7 above)
Comment (zhulinJulia24) [reply benchmark/benchmark_13b.sh:70]: $1已经去掉了，可以忽略

File: docs/en/benchmark/a100_fp16.md (hunk 5 above)
Comment (lvhan028) [reply docs/en/benchmark/a100_fp16.md:9]: 不用传入

File: benchmark/benchmark_13b.sh (hunk 7 above)
Comment (lvhan028) [reply benchmark/benchmark_13b.sh:70]: 没看明白。如果是说可以忽略上面的comments，请"resolve conversation"

**Review by RunningLeon — APPROVED** [review]
LGTM

**Review by zhulinJulia24 — APPROVED** [review]

**Pull request merged**